pairing/bn256/curve.go

```diff
@@ -25,7 +25,7 @@ func (c *curvePoint) String() string {
 	x, y := &gfP{}, &gfP{}
 	montDecode(x, &c.x)
 	montDecode(y, &c.y)
-	return "(" + x.String() + ", " + y.String() + ")"
```

Comment:
I don't have a strong objection against that (and the change in `String`) but it is important to note that this is a **hack** because of the way **onet** works with suite group and encodings. It has nothing to do with kyber. 
So we should not forget that we'll probably have another error of this type in the future, and onet is the place to look for a fix !
I would have preferred a way to fix onet rather than hack kyber but this is clearly something reasonable so I'm fine  with it :)

Reply:
you're right - I had the same feeling. Checking when putting it back, it doesn't crash onet anymore!?! So perhaps this was only needed for an old version. I'll check with @JoaoAndreSa

Reply:
OK, not needed anymore. I put back the 'original' strings.